client/extensions/wp-job-manager/app/controller.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * External Dependencies
+ */
+import React from 'react';
+import i18n from 'i18n-calypso';
+
+/**
+ * Internal Dependencies
+ */
+import analytics from 'lib/analytics';
+import titlecase from 'to-title-case';
+import { getSiteFragment, sectionify } from 'lib/route';
+import { setDocumentHeadTitle as setTitle } from 'state/document-head/actions';
+import { getSelectedSite } from 'state/ui/selectors';
+import { renderWithReduxStore } from 'lib/react-helpers';
+import WPJobManager from './main';
+
+export function settings( context ) {
+	const siteId = getSiteFragment( context.path );
+	const site = getSelectedSite( context.store.getState() );
```

Comment:
Fixed in https://github.com/Automattic/wp-calypso/pull/14972/commits/05937b287ccd7ef6d9b1d045e027555ceafc92aa.